analyzer/codechecker_analyzer/cmd/analyze.py

```diff
@@ -466,6 +466,25 @@ def add_arguments_to_parser(parser):
                                    "same translation unit without "
                                    "Cross-TU enabled.")
 
+        # Only check for AST loading modes if CTU is available.
+        if analyzer_types.is_ctu_on_demand_available(context):
+            ctu_opts.add_argument('--ctu-ast-creation',
+                                  action='store',
+                                  dest='ctu_ast_creation',
+                                  choices=['from-pch', 'parsed-on-demand'],
+                                  default='from-pch',
+                                  help="Choose the way ASTs are loaded during "
+                                       "CTU analysis. Mode 'from-pch' "
+                                       "generates PCH format serialized ASTs "
+                                       "during the 'collect' phase. Mode "
+                                       "'parsed-on-demand' only generates the "
+                                       "invocations needed to parse the ASTs. "
+                                       "Mode 'from-pch' can use significant "
+                                       "disk-space for the serialized ASTs, "
+                                       "while mode 'parsed-on-demand' can "
+                                       "incur some runtime overhead in the "
```

Comment:
runtime memory or CPU or both overhead?

Reply:
only CPU, i will mention it in the comments